src/main/kotlin/no/fdk/fdk_event_harvester/rabbit/RabbitMQConfig.kt

```diff
@@ -0,0 +1,40 @@
+package no.fdk.fdk_event_harvester.rabbit
+
+import org.springframework.amqp.core.*
+import org.springframework.amqp.support.converter.Jackson2JsonMessageConverter
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Configuration
+
+@Configuration
+open class RabbitMQConfig {
+
+    @Bean
+    open fun receiverQueue(): Queue = AnonymousQueue()
```

Comment:
Beanmetodene trenger ikke være open

Reply:
`@Bean-method in @Configuration must be overridable`

Må være open for å være overridable